Home6/Doctor.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Home6
+{
+    internal class Doctor
+    {
+        string Name { get; set; }
+        string Specialization { get; set; }
+
+        internal Doctor()
+        {
+
+        }
+        internal Doctor(string name, string specialization)
+        {
+            Name = name;
+            Specialization = specialization;
+        }
+
+        internal void Treat(Patient patient)
```

Comment:
It is better to use 3 classes and use polymorphism. Please, watch the video recording - lesson 7